Dockerfile

```diff
@@ -8,23 +8,51 @@ ENV COMPOSER_PROCESS_TIMEOUT 3600
 WORKDIR /code/
 
 COPY docker/php-prod.ini /usr/local/etc/php/php.ini
-COPY docker/composer-install.sh /tmp/composer-install.sh
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
         git \
         locales \
         unzip \
+        libaio1 \
 	&& rm -r /var/lib/apt/lists/* \
 	&& sed -i 's/^# *\(en_US.UTF-8\)/\1/' /etc/locale.gen \
-	&& locale-gen \
-	&& chmod +x /tmp/composer-install.sh \
-	&& /tmp/composer-install.sh
+	&& locale-gen
 
 ENV LANGUAGE=en_US.UTF-8
 ENV LANG=en_US.UTF-8
 ENV LC_ALL=en_US.UTF-8
 
+# Oracle instantclient
+RUN mkdir /opt/oracle
+
+COPY docker/oracle/instantclient-basiclite-linux.x64-12.2.0.1.0.zip /opt/oracle/
+COPY docker/oracle/instantclient-sdk-linux.x64-12.2.0.1.0.zip /opt/oracle/
```

Comment:
Oracle ovladace nemozu byt v public repe pokial viem, preto je extractor aj writer private.
Myslim si, ze najnovsi postup je taky, ze sa to hodi na `S3 drivers` a odtial sa to stahuje a repo moze zostat public.

Tak ako je to v `oracle-java-exporter` ten je public: https://github.com/keboola/java-oracle-exporter/blob/02dee88f7964d54d284b3bb4af686d6724e676eb/Dockerfile#L1-L4

Reply:
jo to jsem chtěl dát na S3 ale nějak jsem na to zapomněl... díky přehodím to tam

Reply:
je to tam